src/test/scala/com/cognite/spark/datasource/TimeSeriesRelationTest.scala

```diff
@@ -26,6 +26,21 @@ class TimeSeriesRelationTest extends FlatSpec with Matchers with SparkTest {
 
   val testDataUnit = "time-series-test-data"
 
+  it should "handle pushdown filters on assetIds" taggedAs WriteTest in {
+    val metricsPrefix = "pushdown.filters.assetIds"
+    val df = spark.read
+      .format("com.cognite.spark.datasource")
+      .option("apiKey", readApiKey.apiKey)
+      .option("type", "timeseries")
+      .option("collectMetrics", "true")
+      .option("metricsPrefix", metricsPrefix)
+      .load()
+      .where(s"assetId In(6191827428964450, 3424990723231138, 3047932288982463)")
```

Comment:
Add a test for `assetId in ()` and one where `assetId in (9999)` or other invalid id (should read 0 rows).

Reply:
Good tip! Turns out Spark doesn't like `in()`, so didn't add one for that.